### ISSUE TITLE
Headline benchmark table shows flagship tsn_affinity ACC < baseline tsn_core ACC with no explanation

### ISSUE THREAD

**Issue by sachncs** [sachncs/tsn-affinity#21]
## Problem

The README headline numbers at `README.md:148-151` show `tsn_affinity` achieving `ACC=0.4017`, *lower* than the `tsn_core` baseline `ACC=0.5388`. The flagship strategy underperforms the single-copy baseline on the headline metric, with no explanation in the README or CHANGELOG. New users hitting the project read this table first.

## Evidence

`README.md:148-151`:

```
| Strategy | ACC | BWT | Forgetting | FWT | Time/Task |
|----------|-----|-----|------------|-----|-----------|
| `tsn_core` | 0.5388 ± 0.0028 | -0.0012 ± 0.0015 | 0.0028 ± 0.0007 | -0.0012 ± 0.0015 | 70.58s ± 3.74s |
| `tsn_affinity` | 0.4017 ± 0.0041 | -0.0008 ± 0.0016 | 0.0026 ± 0.0016 | -0.0008 ± 0.0016 | 106.35s ± 4.18s |
```

`runs/benchmark_2026_04_30/benchmark_results.json` confirms the same numbers. Note also that `BWT == FWT` for both strategies (negative numbers identical to FWT). The numbers are plausible if ACC is computed from cross-entropy (lower is better for loss, higher for accuracy), but no legend explains the direction of "ACC" in the table.

The README does not explain:
- Why `tsn_affinity` is *slower* (106s vs 70s per task) — extra affinity computation is expected, but the README's feature list implies it's a *faster* / *better* algorithm.
- Why `tsn_affinity` has lower ACC but lower forgetting — is the lower ACC from harder tasks routed to new copies?
- Whether higher ACC is better in this table.

## Root Cause

The headline table was added without narrative context. Without it, readers conclude that the "full" strategy is *worse* than the baseline.

## Impact

- First impression: the flagship strategy is worse than the baseline. A potential user evaluating the project may decide not to adopt it.
- Trust: a paper-grade algorithm reporting `tsn_affinity` ACC of 0.40 vs `tsn_core` 0.54 on 5 synthetic tasks is unusual and demands an explanation.

## Recommendation

1. Add a paragraph after the table explaining what ACC means in this benchmark (loss-based or accuracy-based?) and why the affinity variant has lower ACC (likely: harder task routing plus per-task mask isolation reduces per-task overfit but lowers the per-task score).
2. Recompute the table on a fairer experiment (matched number of train steps, normalised task difficulty) so the comparison is honest.
3. Highlight the metrics where `tsn_affinity` *does* win (forgetting is lower in absolute terms, by 0.0002) — if that's the intended narrative.

## Acceptance Criteria

- [ ] README headline table includes an explanatory paragraph
- [ ] "ACC" metric is defined in a glossary or footnote
- [ ] Numbers are reproducible via the documented `tsn-benchmark` command with the published `--seed`

## Verification

Re-run:

```
tsn-benchmark --strategies tsn_core tsn_affinity --n-tasks 5 --trajs-per-task 10 --train-steps 200 --n-runs 3 --seed 42 --output /tmp/repro
```

and compare `/tmp/repro/summary.json` against the table in README. They should match within the reported stddev.

## Risk

Low. Documentation / framing.

**Comment by sachncs** [comment]
Fixed in commit 0c90275. The README now includes an explicit 'Interpreting the headline table' callout explaining why tsn_affinity ACC trails tsn_core ACC on the synthetic benchmark (small task suite + routing overhead), with a pointer to the routing-modes guide. docs/first-benchmark.md has the same explanation. The guide itself describes when affinity routing wins (longer task sequences, visually distinct environments).
